src/components/toast/toast.js

```diff
@@ -170,7 +170,7 @@ export const BToast = /*#__PURE__*/ Vue.extend({
     },
     toaster(newVal) /* istanbul ignore next */ {
       // If toaster target changed, make sure toaster exists
-      this.$nextTick(() => this.ensureToaster)
+      this.$nextTick(() => this.ensureToaster())
```

Comment:
This actually could be written as:

```js
this.$nextTick(this.ensureToaster)
```

Reply:
But wouldn't that change the scope?

Reply:
All methods of a component are automatically bound to `this`